azavea/raster-foundry
# Feature/lk/reduce grid refreshes

## Overview

Refresh grid only when required

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Notes

Depends on #1363 

## Testing Instructions

* Go to the Browse view
* For each filter, verify that changing/disabling/enabling the filter will cause the grid to reload, but panning the map does not

Connects #1360
